### ISSUE TITLE
Tab and spacebar keyboard-accessibility don't work on Widevine panel

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#14578]
## Description 
Tab and spacebar keyboard-accessibility don't work on Widevine panel.  This might be a more-general/widespread problem - I did search for duplicate bugs, though.

## Steps to Reproduce

   1.  new profile, launch Brave
   2. load `https://bitmovin.com/demos/drm`
   3. when the Widevine popup appears, try pressing `tab` to forward-cycle through the browser's UI elements
   4. click the "Don't ask again" checkbox
   5. now press the `spacebar`
   6. now press `tab` again

## Actual results:

3. `tab`ing doesn't do anything
4. there's no focus ring around the checkbox and its text
5. nothing happens
6. nothing happens

<img width="1312" alt="Screen Shot 2021-03-08 at 4 40 24 PM" src="https://user-images.githubusercontent.com/387249/110400731-09216800-802d-11eb-87d7-6c19ebc812cd.png">

## Expected results:
3. pressing `tab` should forward-cycle (using tab-index) through UI elements, focusing them with each press
4. there should be a focus ring around the checkbox and its text
5. pressing the `spacebar` should toggle the checked/unchecked state of the "Don't ask again" checkbox
6. tabbing from the `Don't ask again` checkbox should logically focus the `Block` button element, I'd think

## Brave version (brave://version info)

Brave | 1.23.23 Chromium: 89.0.4389.72 (Official Build) nightly (x86_64)
-- | --
Revision | 3f345f156bfd157bd1bea06310e55f3fb2490359-refs/branch-heads/4389@{#1393}
OS | macOS Version 11.2.3 (Build 20D91)

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the nightly channel? yes

cc: @karenkliu
